docs/platform-engineers/addon/intro.md

```diff
@@ -2,48 +2,122 @@
 title: Customize Addon
 ---
 
-Addon can be used to manage and extend KubeVela platform capabilities.
-Users and install/uninstall addons on KubeVela via UI/CLI interfaces.
-Addons can be stored and discovered via addon registries.
+The picture below shows what KubeVela does when an addon is enabled. You can see that the Addon Registry stores the addon's resource files. When an addon is enabled through UX/CLI, they will pull these resource files from the Addon Registry, render them and create a KubeVela application. Finally, the KubeVela controller running in the management cluster completes the delivery of the resources described in the application.
```

Comment:
```suggestion
The picture below shows what KubeVela does when an addon is enabled. You can see that the Addon Registry stores the addon's resource files. When an addon is enabled through UX/CLI, it will pull these resource files from the Addon Registry, render them and create a KubeVela application. Finally, the KubeVela controller running in the management cluster completes the delivery of the resources described in the application.
```